utils/docker/Dockerfile.centos.7

```diff
@@ -97,6 +97,7 @@ RUN dnf -y install deltarpm && \
     dnf -y upgrade && \
     dnf -y install \
         boost-python36-devel \
+        bzip2 \
```

Comment:
I guess this is the simplest fix, but is there a reason why our implementation and interaction between `utils/docker/Dockerfile.centos.7` and `utils/scripts/install-centos7.sh` differs so much between the master and release/2.0 branches?  Since the `utils/scripts/install-centos7.sh` exists in the release/2.0 branch (and it already includes `bzip2`) would it make more sense for this file to implement calling it  https://github.com/daos-stack/daos/blob/master/utils/docker/Dockerfile.centos.7#L89-L92 like we do in master or is the release/2.0 approach the direction we want to take (where we don't need the `utils/scripts/install-centos7.sh` file)?

Reply:
We've done a lot of cleanup of docker on master recently which explains the difference, none of it was intended to be backported as far as I'm aware.  It looks like the `utils/scripts/install-centos7.sh` script on release/2.0 but have been included by accident.